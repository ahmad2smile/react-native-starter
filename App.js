import React, { Component } from "react"
import { Provider } from "react-redux"

import store from "./appstate/store/"

import NavigationComponent from "./navigation/NavigationComponent"

export default class App extends Component {
	render () {
		return (
			<Provider store={store}>
				<NavigationComponent />
			</Provider>
		)
	}
}
