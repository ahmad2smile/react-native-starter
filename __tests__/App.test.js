import React from "react"
import "react-native"
import { Provider } from "react-redux"
import configureStore from "redux-mock-store"

// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer"
import App from "../App"

// import { initialState as login } from "../../../appstate/reducers/login/loginReducer"

const initialState = {}

const store = configureStore()(initialState)
it("renders correctly", () => {
	const wrapper = renderer.create(<App store={store} />)
	expect(wrapper).toMatchSnapshot()
})
