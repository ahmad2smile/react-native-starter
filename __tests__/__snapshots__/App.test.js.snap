// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(0, 0, 0, 0.3)",
      "flex": 1,
      "height": "100%",
      "justifyContent": "center",
      "marginHorizontal": 0,
      "position": "absolute",
      "width": "100%",
      "zIndex": 3,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#88b04b"
    hidesWhenStopped={true}
    size="small"
  />
</View>
`;
